src/mscorlib/shared/System/Globalization/DateTimeFormat.cs

```diff
@@ -852,9 +853,15 @@ private static void FormatCustomizedRoundripTimeZone(DateTime dateTime, TimeSpan
                 offset = offset.Negate();
             }
 
-            AppendNumber(result, offset.Hours, 2);
+            Append2DigitNumber(result, offset.Hours);
             result.Append(':');
-            AppendNumber(result, offset.Minutes, 2);
+            Append2DigitNumber(result, offset.Minutes);
+        }
+
+        private static void Append2DigitNumber(StringBuilder result, int val)
+        {
+            result.Append((char)('0' + (val / 10)));
+            result.Append((char)('0' + (val % 10)));
```

Comment:
Use the DivMod helper method that we use in Utf8Formatter?

![image](https://user-images.githubusercontent.com/6527137/37728241-f7b89b92-2cf6-11e8-9bfe-f3b5b24af9bc.png)

```C#
private static void Append2DigitNumber(StringBuilder result, int val)
{
    result.Append((char)('0' + (val / 10)));
    result.Append((char)('0' + (val % 10)));
}

private static void Append2DigitNumber2(StringBuilder result, int val)
{
    uint div = DivMod((uint)val, out uint modulo);
    result.Append((char)('0' + div));
    result.Append((char)('0' + modulo));
}

[MethodImpl(MethodImplOptions.AggressiveInlining)]
public static uint DivMod(uint numerator, out uint modulo)
{
    uint div = numerator / 10;
    modulo = numerator - (div * 10);
    return div;
}

```

Reply:
> Use the DivMod helper method that we use in Utf8Formatter?

I can switch to Math.DivRem.